test/spec/invertedIndexSpec.js

```diff
@@ -32,61 +26,55 @@ describe('InvertedIndex', () => {
       expect(InvertedIndex.validateFile(notArray)).toBeFalsy();
       expect(InvertedIndex.validateFile(wrongData)).toBeFalsy();
     });
-    it('should return the file content if a valid file was uploaded', () => {
-      expect(InvertedIndex.validateFile(validFile)).not.toBeFalsy();
-      expect(InvertedIndex.validateFile(simpleValidFile)).not.toBeFalsy();
+
+    it('should not return false if a valid file was uploaded', () => {
+      expect(InvertedIndex.validateFile(books)).not.toBeFalsy();
+      expect(InvertedIndex.validateFile(shortStories)).not.toBeFalsy();
+      expect(typeof InvertedIndex.validateFile(books)).toEqual('object');
```

Comment:
I think this can be moved to another test case which checks that it returns an Object if a valid file it uploaded.